lib/zendesk_apps_support/location.rb

```diff
@@ -0,0 +1,29 @@
+module ZendeskAppsSupport
+  module Location
+    # the numbers below match the enum values on the database, do not change them!
```

Comment:
I wondered about that to, couldn't think of any problems with leaving it. We definitely need a warning of some sort.